Packs/QRadar/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IBM QRadar v2
+  - Simplified the way asset properties are displayed in fetched incident. This may adversely affect existing assets mapping.
+  - Fixed an issue where the time field occurred in fetched incidents stored timestamp value instead of an ISO format time string.
```

Comment:
```suggestion
  - Fixed an issue where the time occurred field for fetched incidents would store a timestamp value instead of an ISO formatted time string.
```